lambda/create-investigation/handler.py

```
@@ -155,12 +192,13 @@ def lambda_handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
             task_timeout = int(task_timeout)
             if task_timeout < 0 or task_timeout > 86400:
                 raise ValueError("Task timeout out of range")
-            if TASK_TIMEOUT_MINIMUM > 0 and task_timeout < TASK_TIMEOUT_MINIMUM:
+            # Allow task_timeout=0 (means no timeout), enforce minimum only for positive values
+            if TASK_TIMEOUT_MINIMUM > 0 and task_timeout > 0 and task_timeout < TASK_TIMEOUT_MINIMUM:
                 raise ValueError(f"task_timeout below minimum ({TASK_TIMEOUT_MINIMUM}s)")
         except (ValueError, TypeError) as e:
             logger.warning(f"Invalid task_timeout: {task_timeout} — {str(e)}")
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)